int16 m_ang
float32 s1_dist
int16 s1_ang
float32 s2_dist
int16 s2_ang
float32 s3_dist
int16 s3_ang


int16 recog_ang
int16 sig_timing
int16 real_ang
float32 real_dist